README.adoc

```diff
@@ -296,28 +296,28 @@ include::finish/src/main/java/io/openliberty/guides/beanvalidation/Astronaut.jav
 ---- 
 
 BeanValidationEndpoint.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/beanvalidation/BeanValidationEndpoint.java[tags=**;!copyright]
+include::finish/src/main/java/io/openliberty/guides/beanvalidation/BeanValidationEndpoint.java[]
 ----
 
-Two resources, a validator and an instance of the [hotspot=27 file=2]`Spacecraft` JavaBean, 
+Two resources, a validator and an instance of the [hotspot=Spacecraft file=2]`Spacecraft` JavaBean, 
 are injected into the class. The default validator is used and is obtained through CDI
 injection. However, you can also obtain the default validator with resource injection
-or a JNDI lookup. The [hotspot=17-63 file=0]`Spacecraft` JavaBean is injected so that the 
+or a JNDI lookup. The [hotspot=Spacecraft file=0]`Spacecraft` JavaBean is injected so that the 
 method-level constraints can be validated.
 
-The programmatic validation takes place in the [hotspot=34-54 file=2]`validateSpacecraft` 
-method. To validate the data, the [hotspot=42 file=2]`validate` method is called on the 
-[hotspot=17-63 file=0]`Spacecraft` bean. Because the [hotspot=17-63 file=0]`Spacecraft` 
-bean contains the [hotspot=21 file=0]`@Valid` constraint on the [hotspot=11-50 file=1]`Astronaut` 
+The programmatic validation takes place in the [hotspot=validate-Spacecraft file=2]`validateSpacecraft` 
+method. To validate the data, the [hotspot=validate file=2]`validate` method is called on the 
+[hotspot=Spacecraft file=0]`Spacecraft` bean. Because the [hotspot=Spacecraft file=0]`Spacecraft` 
+bean contains the [hotspot=Valid file=0]`@Valid` constraint on the [hotspot=Astronaut file=1]`Astronaut` 
 bean, both JavaBeans are validated. Any constraint violations found during the call to 
-the [hotspot=42 file=2]`validate` method are returned as a set of [hotspot=41 file=2]`ConstraintViolation` 
+the [hotspot=validate file=2]`validate` method are returned as a set of [hotspot=ConstraintViolation file=2]`ConstraintViolation` 
 objects.
 
-The method level validation occurs in the [hotspot=60-71 file=2]`launchSpacecraft` method. 
-A call is then made to the [hotspot=58-62 file=0]`launchSpacecraft` method on the 
-[hotspot=17-63 file=0]`Spacecraft` bean, which throws a `ConstraintViolationException` 
+The method level validation occurs in the [hotspot=launchSpacecraft file=2]`launchSpacecraft` method. 
+A call is then made to the [hotspot=launch-Spacecraft file=0]`launchSpacecraft` method on the 
```

Comment:
<img width="1606" alt="image" src="https://user-images.githubusercontent.com/8294670/68787453-90b0bf80-060f-11ea-9f59-d7e3ff84a798.png">

Reply:
launchSpacecraft hotspot not working correctly.

Reply:
Updated.